compiler/rustc_session/src/session.rs

```diff
@@ -732,6 +732,10 @@ impl Session {
         self.opts.cg.split_debuginfo.unwrap_or(self.target.split_debuginfo)
     }
 
+    pub fn dwarf_version(&self) -> u32 {
```

Comment:
Maybe worth a little comment like `/// Returns the dwarf-version passed on the CLI or the target's default version` ?